### PR TITLE
Tests Update README.md and DOCKER.md

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | add functional tests to ReadMe
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | No need to test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16659)

### CODE REVIEW

**Review by SimonGrn — DISMISSED** [review]

File: tests/puppeteer/DOCKER.md [hunk 1]
```diff
@@ -16,6 +16,15 @@ docker-compose -f docker-compose.nightly.yml -f docker-compose.tests.yml exec -e
 docker-compose -f docker-compose.nightly.yml -f docker-compose.tests.yml up --build
 docker-compose -f docker-compose.nightly.yml -f docker-compose.tests.yml exec -e COMMAND="sanity-tests" tests bash /tmp/run-tests.sh
 ```
+
+### Functional tests
+
+```bash
+# Create Shop and running test
```
Comment (PierreRambaud) [inline tests/puppeteer/DOCKER.md]: Move this comment into the markdown part

File: tests/puppeteer/README.md [hunk 2]
```diff
@@ -53,8 +53,19 @@ If you want to run all sanity tests "safely", you can use the Travis-specific co
 npm run sanity-travis
 ```
 
+## Functional tests 
+This campaign verifies that each function of the software application, operates in conformance with the functional requirements.
```
Comment (PierreRambaud) [inline tests/puppeteer/README.md]: Functions are operating, not the campaign

```suggestion
This campaign verifies that each function of the software application operate in conformance with the functional requirements. 
```

Comment (boubkerbribri) [reply tests/puppeteer/README.md]: done thanks

File: tests/puppeteer/DOCKER.md (hunk 1 above)
Comment (boubkerbribri) [reply tests/puppeteer/DOCKER.md]: done thanks

**Review by PierreRambaud — DISMISSED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**